docs/changehistory/NextVersion.md

```diff
@@ -28,7 +28,27 @@ Can now be expressed as follows (note no IModelConnection is required):
   });
 ```
 
-## Model map layers
+## Map Layers
+
+### MapLayersUI initialization
+
+In previous versions, MapLayersUI was intialized with an ever growing list of unnamed args. Now it is initialized with a single [MapLayersConfig]($map-layers) object with named args. The following code snippet demonstrates how to initialize MapLayersUI:
```

Comment:
Convert "MapLayersUI" into a docs link; note that it's a `@beta` API since you're breaking it.

Reply:
Actually...this is an Extension. It doesn't have an API, does it?

Reply:
tbh im not sure how i feel about this being called an "Extension". Eventually we'd want this to be built with the extension api, but atm it doesnt and its misleading. 

I didnt even realize we never had any docs for map-layers :/

Reply:
Yeah, it's weird. ui-test-app consumes it as a package and uses its API. It has an empty docs script, no extract-api script, yet (some) APIs have release tags.
Perhaps it should not be mentioned in NextVersion.md at all until it's been sorted out.

Reply:
Thanks for calling this out @pmconne ; I made sure to add an AHA item for "extension" documentation to tackle in the coming weeks. Reverted my changes here until we have a solution.